### DISCUSSION TITLE
⏳ 🔗 Allow to set a Fallback Image for the Category Image Dynamic Tag

### DISCUSSION THREAD

**Discussion by micronzone** [elementor/elementor#25703] [Feature Request]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**

**Describe the solution you'd like**
1. Fallback Image cannot be specified in Dynamic Tags> Category Image.
Since a blank image is output from an Archive type page, you need to be able to create a Fallback Image.

2. I wish I could insert Body Class in Template Builder.

**Describe alternatives you've considered**

**Additional context**

**Comment by kitsunecreativeco** [comment]
Another + 1 for this - there really needs to be an image fallback option for dynamic category images as there are for post images etc

**Comment by nicholaszein** [comment]
Hi there,

### ✈️ This Feature Request has been migrated from issue #13614

if you still think this request should be considered for our roadmap, please cast your **⬆️ Upvote** on the first comment.

Happy voting! 🙂

**Comment by Daveden2** [comment]
This feature might finally get more traction when Elementor's Taxonomy Query gets released in 3.21 (for WooCommerce users).

**Comment by awebguy99** [comment]
+1 100% needs this.  Why doesn't every dynamic field have a fallback option?
